astor/rtrip.py

```diff
@@ -85,6 +85,12 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s):
+    if b'' == u'':
```

Comment:
This would raise a `BytesWarning` if you invoke `python` with `-b` (which a lot of people use it to avoid some weird bugs in their code). Perhaps we can use `io.open(..., encoding='utf-8')` instead. I don't remember if it's available in Python 2.6 though. If it doesn't, maybe it's time to finally drop Python 2.6 support? :)